build/packaging/rpm/glideinwms.spec

```diff
@@ -1106,6 +1106,11 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %changelog
+* Fri Jan 24 2025 Marco Mambelli <marcom@fnal.gov> - 3.10.10
+- Glideinwms v3.10.10
+- Release Notes: http://glideinwms.fnal.gov/doc.v3_10_10/history.html
+- Release candidates 3.10.10-01.rc1
```

Comment:
Do we have a release candidate for 3.10.10? From reading the description in the PR, it seemed like this would directly be a release without RCs. Can you please confirm?

Reply:
I did an RC on the testbed. No outside release